docker/Dockerfile.base

```diff
@@ -1,25 +1,29 @@
-ARG CUDA_VERSION=11.4.0-cudnn8-runtime-ubuntu20.04
+ARG CUDA_VERSION
 FROM nvidia/cuda:${CUDA_VERSION}
 
-ARG DEEPLABCUT_VERSION=2.2.1.1
+ARG DEEPLABCUT_VERSION
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update -yy \ 
-    && apt-get install -yy --no-install-recommends python3 python3-pip python3-dev ffmpeg libsm6 libxext6 \
+    && apt-get install -yy --no-install-recommends python3.8 python3-pip python3-dev ffmpeg libsm6 libxext6 \
```

Comment:
Why not a more recent python version?

Reply:
Our deployed conda env is 3.9; there should be no need to pin numpy in the Docker, it's set in dlc setup.py ?

Reply:
I was following the ensemble of dependencies that have already existed  in the initial Dockerfile, and eventually I would like to get rid off of any duplicates (from setup etc), so once we have a one stable version, I was testing, what could we throw away. Concerning numpy, I think that the version of numpy is precised explicitly, as in the dlc setup.py it's >=1.18.5, however how we can notice, it's working only with 1.21 without any dramatic conflicts from tensorflow for example. @jeylau what do you think

Reply:
Yes, it's the case: once I don't precise the version it installs 1.24.4 and during the dlc import the error "AttributeError: module 'numpy' has no attribute 'object'." appears. (it's not dlc itself, but TensorFlow 2.5 side), but as was discussed earlier we can't upgrade the tf version, as cuda (11.4)